src/lucky/action_callbacks.cr

```diff
@@ -24,14 +27,64 @@ module Lucky::ActionCallbacks
     \{% end %}
   end
 
+  # Run a method before an action starts
+  #
+  # Methods will run in the order that each `before` is defined. Also, each
+  # method must call `redirect`, `render`, or `continue`:
```

Comment:
Actually just realized this may need to be clarified. Smething like: 

> the callback must return a `LuckyWeb::Response` using `render`, `redirect`, `json`, etc., or if you don't want to render anything `continue`

This is horribly worded, but the idea is that it says it must return a `LuckyWeb::Response`. That could be with render, redirect, json, head, xml or anything else like that. I don't want people to get confused and think they can only render or redirect. Does that make sense?

Reply:
Yeah, I gotcha. I'll futz with it